### DISCUSSION TITLE
Local Tuya

### DISCUSSION THREAD

**Discussion by ShortPutt** [tuya/tuya-home-assistant#853] [General]
Is there any word on when the Home Assistant Tuya integration will support local / cloudless access to Tuya devices? Last year I heard this was on their roadmap, but haven't heard anything about it since.

**Comment by Skid-Br0** [comment]
bump, any updates on this??

**Comment by ShortPutt** [comment]
Last I heard Tuya shelved the project. My guess is its not going to happen.
I'm using the cloud integration, but plan to flash tasmota or ESPHome when I get the time

**Reply by Skid-Br0** [reply]
That's a shame, makes sense though. Half the reason the bulbs are so cheap is probably from the data they harvest. Iv gone with Tuya Local, Seems to work okay so far
